src/libraries/System.Runtime/tests/System.Runtime.Tests/System/Runtime/CompilerServices/RuntimeHelpersTests.cs

```diff
@@ -437,6 +437,32 @@ public static void FixedAddressValueTypeTest()
 
             Assert.Equal(fixedPtr1, fixedPtr2);
         }
+
+        [InlineArray(4)]
+        private struct Byte4
```

Comment:
```suggestion
        [InlineArray(3)]
        private struct Byte3
```
3 byte would be more interesting - to ensure that the size is not getting aligned up.